### PR TITLE
Run bundle check before running bundle in case bundle has already been run

### PR DESCRIPTION
**Pull request by jeckhart**
Per suggestion in #94

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
What's the advantage/use-case of this?

**Comment by jeckhart** [comment]
My dotfiles run `brew bundle` during their bootstrap. The `brew bundle` at the end of strap is duplicative in this case so this skips it if it's already been run.

File: bin/strap.sh [hunk 1]
```diff
@@ -318,7 +318,7 @@ fi
 # Install from local Brewfile
 if [ -f "$HOME/.Brewfile" ]; then
   log "Installing from user Brewfile on GitHub:"
-  brew bundle --global
+  brew bundle check --global >/dev/null 2>&1 || brew bundle --global
```
Comment (MikeMcQuaid) [inline bin/strap.sh]: I think the `>/dev/null 2>&1` should be removed here as we want to see the `The Brewfile's dependencies are satisfied.` when this is run.

**Comment by MikeMcQuaid** [comment]
@jeckhart Thanks for your first contribution to Strap. You rock!

**Comment by jeckhart** [comment]
Absolutely, and thanks for strap!

**Pull request merged**